src/xsl/openrosa2html5form.xsl

```diff
@@ -105,13 +105,13 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                             <xsl:value-of select="concat(' ', /h:html/h:body/@class)" />
                         </xsl:if>
                     </xsl:attribute>
-                    <xsl:attribute name="id">
+                    <xsl:attribute name="form-id">
```

Comment:
In the interest of HTML validity/forward-compatibility, and consistency with similar attribute transformations, I think it's safer to use `data-form-id`. The `data-` prefix is the namespace for arbitrary custom attributes, and will also populate `form.dataset` for consistent access.